src/components/Author.jsx

```diff
@@ -1,20 +1,17 @@
 import React from 'react';
 
 export default class Author extends React.Component {
+    // TODO: establish static JSON object for authors
+    // structure: {name, photo, biography, articleKeys=[]}
```

Comment:
for articleKeys, look into use the `ref` object given by Falcor. It lets you reference another part of the Falcor graph without having to duplicate the data.